Library/Formula/tesseract.rb

```diff
@@ -128,6 +128,10 @@ class Tesseract < Formula
     sha1 "a559801a42be8a3dc15ae2ec27da99bd8b0b85ac"
   end
 
+  resource "tessdata-head" do
```

Comment:
Let's tuck this inside the `head do` block just to make it clear it only applies to the `head` logic. You should be able to drop the `resources.reject { |r| r.name == "tessdata-head" }.each` dance below as well then.